src/main/java/cz/cvut/kbss/termit/dto/PasswordChangeDto.java

```diff
@@ -0,0 +1,41 @@
+package cz.cvut.kbss.termit.dto;
+
+import java.io.Serializable;
+import java.net.URI;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+public class PasswordChangeDto implements Serializable {
+    private URI uri;
+    private String token;
+    private String newPassword;
+
+    public URI getUri() {
+        return uri;
+    }
+
+    public void setUri(URI uri) {
+        this.uri = uri;
+    }
+
+    public void setUri(String uri) {
```

Comment:
What is the use of this method? This class is a DTO, so its getters and setters should be as simple as possible. And if they do contain such logic as URL decoding, it should be properly documented.

Reply:
Ok, I didn't even find any uses of this method!